Dockerfile.template

```diff
@@ -93,6 +93,11 @@ COPY --from=composer:{{ .composer.version }} /usr/bin/composer /usr/local/bin/
 ENV DRUPAL_VERSION {{ .version }}
 {{ if has("composer") then ( -}}
 
+# https://github.com/moby/buildkit/issues/4503
```

Comment:
Now that we have a docker-library/drupal number, we should maybe inject it too :eyes:

```suggestion
# https://github.com/docker-library/drupal/pull/259
# https://github.com/moby/buildkit/issues/4503
```